src/browser-pool.ts

```diff
@@ -2,12 +2,19 @@ import pLimit from 'p-limit';
 import { nanoid } from 'nanoid';
 import ow from 'ow';
 import { TypedEmitter } from 'tiny-typed-emitter';
+import { FingerprintInjector } from 'fingerprint-injector';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore bypass unnecessary module declaration for tests
+import FingerprintGenerator from '../node_modules/fingerprint-generator';
```

Comment:
better to use `ts-expect-error` so you get "notified" (via failed build) if the behaviour changes (so here once the package has types)

```suggestion
// @ts-expect-error no types for this package yet
import FingerprintGenerator from 'fingerprint-generator';
```